browser/profiles/BUILD.gn

```diff
@@ -1,3 +1,4 @@
+import("//brave/components/decentralized_dns/buildflags/buildflags.gni")
```

Comment:
?

Reply:
I needed to add this import **not** to get the following error:
```
> gn check /home/mario/work/brave-browser/src/out/Component //brave/*
ERROR at //brave/browser/profiles/BUILD.gn:65:7: Undefined identifier
  if (decentralized_dns_enabled) {
      ^------------------------
See //brave/browser/BUILD.gn:120:5: which caused the file to be included.
    "profiles",
    ^---------
```

And the reason this is needed now is because of removing the `import("//build/config/features.gni")` line from `components/tor/buildflags/buildflags.gni`:

Previous to this patch, `brave/browser/profiles/BUILD.gn` would indirectly satisfy the dependency on `//brave/components/decentralized_dns/buildflags` via [the `//brave/components/tor/buildflags/buildflags.gni` import it already has in place](https://github.com/brave/brave-core/blob/master/browser/profiles/BUILD.gn#L1), which would end up importing that GNI file as it can be seen in the tree below:
```
brave/browser/profiles/BUILD.gn
└─> brave/components/tor/buildflags/buildflags.gni
     └─> build/config/features.gni
          └─> build/config/chrome_build.gni
               └─> brave/build/config/brave_build.gni
                    └─> brave/net/sources.gni
                         └─> brave/components/decentralized_dns/buildflags/buildflags.gni
```
... but now that the `brave/components/tor/buildflags/buildflags.gni ─> build/config/features.gni` link does not exist we need to explicitly add the import to `brave/browser/profiles/BUILD.gn`, or we'll get a new `gn check` failure.

Sorry if this was not clear enough from the PR.

Reply:
Note: after the latest changes I made, described in https://github.com/brave/brave-core/pull/8443#discussion_r608489705, it's not really strictly needed so I've removed this bit as well from this PR and will incorporate it in the new one I'll propose to get rid of those import from all the affected `buildflags.gni` files at once.